hudi-common/src/main/java/org/apache/hudi/common/table/log/block/HoodieHFileDataBlock.java

```diff
@@ -168,8 +170,14 @@ protected <T> ClosableIterator<HoodieRecord<T>> deserializeRecords(byte[] conten
     // Get schema from the header
     Schema writerSchema = new Schema.Parser().parse(super.getLogBlockHeader().get(HeaderMetadataType.SCHEMA));
 
+    HoodieLogBlockContentLocation blockContentLoc = getBlockContentLocation().get();
+    Configuration inlineConf = new Configuration(blockContentLoc.getHadoopConf());
+    inlineConf.set("fs." + InLineFileSystem.SCHEME + ".impl", InLineFileSystem.class.getName());
+    inlineConf.setClassLoader(InLineFileSystem.class.getClassLoader());
+
+    FileSystem fs = FSUtils.getFs(pathForReader.toString(), inlineConf);
```

Comment:
Should we guard this in HBase version check? Intention is to avoid additional call if fs is not needed in some hbase version.

Reply:
@codope I think its call time is actually very small, and I haven't seen the usage of setting fs to null elsewhere, which is a bit confusing.
Of course, I can also add the version as a condition, what do you think?

Reply:
In `#lookupRecords`, there is similar logic for generating the fs, can we abstract the code snippet into a common utitlies method like `#getFs` ?

Reply:
@danny0405 Thanks, I create a common method `buildInlineConf()`

Reply:
Sounds good @Zouxxyy , as along as we are ok with compatibility.